channels/de.m3u

```diff
@@ -339,129 +339,71 @@ https://hls1.wtnet.de/noa4hh/apple/wifi6500.m3u8
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/54.m3u8
 #EXTINF:-1 tvg-id="NRWision.de" tvg-name="NRWision" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/RkF2Kmx.png" group-title="",NRWision
-https://fms.nrwision.de/live/livestreamHD.stream_1080p/chunklist.m3u8
+https://fms.nrwision.de/live/livestreamHD.stream_1080p/playlist.m3u8
 #EXTINF:-1 tvg-id="NRWision.de" tvg-name="NRWision" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/RkF2Kmx.png" group-title="",NRWision
-https://fms.nrwision.de/live/livestreamHD.stream_360p/chunklist.m3u8
-#EXTINF:-1 tvg-id="NRWision.de" tvg-name="NRWision" tvg-country="DE" tvg-language="German" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/nrwision.png" group-title="",NRWision (360p)
 https://fms.nrwision.de/live/livestreamHD.stream_360p/playlist.m3u8
 #EXTINF:-1 tvg-id="NRWISION.de" tvg-name="NRWISION" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",NRWISION
-https://fms.nrwision.de/live/livestreamHD.stream_source/chunklist_w1070249849.m3u8
-#EXTINF:-1 tvg-id="NRWisionHD.de" tvg-name="NRWision HD" tvg-country="DE" tvg-language="" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/nrwision.png" group-title="",NRWision HD
-https://fms.nrwision.de/live/livestreamHD.stream_source/chunklist.m3u8
-#EXTINF:-1 tvg-id="NrwisionTVHD.de" tvg-name="Nrwision TV HD" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",Nrwision TV HD (1080p)
+https://fms.nrwision.de/live/livestreamHD.stream_source/playlist.m3u8
+#EXTINF:-1 tvg-id="NRWision.de" tvg-name="NRWision" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/RkF2Kmx.png" group-title="",NRWision
 https://fms.nrwision.de/live/livestreamHD.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="OberlausitzTV.de" tvg-name="Oberlausitz TV" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",Oberlausitz TV (1080p)
-http://5856e1a25f71a.streamlock.net:1935/easycast8-live/_definst_/mp4:livestreamhd4/playlist.m3u8
+http://5856e1a25f71a.streamlock.net:1935/easycast8-live/mp4:livestreamhd4/playlist.m3u8
 #EXTINF:-1 tvg-id="OberpfalzTV.de" tvg-name="Oberpfalz TV" tvg-country="DE" tvg-language="German" tvg-logo="" group-title="",Oberpfalz TV (1080p)
 https://oberpfalztv.iptv-playoutcenter.de/oberpfalztv/oberpfalztv.stream_1/playlist.m3u8
 #EXTINF:-1 tvg-id="oeins.de" tvg-name="oeins" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/NHvYg7e.png" group-title="",oeins (Oldenburg)
 https://www.oeins.de/live/studio.m3u8
 #EXTINF:-1 tvg-id="OFTVOffenbach.de" tvg-name="OF-TV Offenbach" tvg-country="DE" tvg-language="" tvg-logo="https://i.imgur.com/BmJaDT3.jpg" group-title="",OF-TV Offenbach (720p)
-https://5864df9ceac85.streamlock.net/germanpictures-live/_definst_/mp4:streamschedule/playlist.m3u8?&ref=www.of-tv.de&seid=166119
```

Comment:
Make sure we have all channels from that large chunk reinstated. I think some of them are missed below

Reply:
Let me see that...

Reply:
By comparing both versions (master and PR), I've got no particular differences, what PR are you thinking about ?

Reply:
Aren't Flensburg, Gießen, Kaiserslautern, Kassel and Kiel a different streams?

Reply:
Different regions, so different channels indeed.

The problem with this playlist is... there's dupes of channels that are below, named differently, and lots of dupes based on the chunks and query string additions.

Reply:
And you're right. Reinstating those. Was a bit frustrated to see so many dupes packed into one place when the majority of the OK channels were below.